build.sbt

```diff
@@ -1468,7 +1462,15 @@ lazy val runtime = (project in file("engine/runtime"))
       .value
   )
   .settings(
-    bench := (Benchmark / test).tag(Exclusive).value,
+    bench := (Benchmark / test)
+      .tag(Exclusive)
+      .dependsOn(
+        // runtime.jar fat jar needs to be assembled as it is used in the
+        // benchmarks. This dependency is here so that `runtime/bench` works
+        // after clean build.
+        LocalProject("runtime-with-instruments") / assembly
+      )
+      .value,
```

Comment:
This fixes [Benchmark Engine action](https://github.com/enso-org/enso/actions/workflows/engine-benchmark.yml) - example of the last failing one is at https://github.com/enso-org/enso/actions/runs/6923737651